{{cookiecutter.project_name}}/contracts/NFT.vy

```diff
@@ -132,17 +126,25 @@ EIP712_DOMAIN_VERSIONHASH: constant(bytes32) = keccak256("1")
 {%- if cookiecutter.metadata == 'y' %}
 NAME: constant(String[20]) = "{{cookiecutter.token_name}}"
 SYMBOL: constant(String[5]) = "{{cookiecutter.token_symbol}}"
-IDENTITY_PRECOMPILE: constant(address) = 0x0000000000000000000000000000000000000004
+{%- endif %}
+
+{%- if cookiecutter.updatable_uri == 'y' %}
+baseURI: public(String[100])
+
+{%- else %}
+BASE_URI: constant(String[100]) = "dummy uri"
```

Comment:
Would love it to add another field in cookiecutter (set to the "coming soon" metadata IPFS CID by default)
```suggestion
BASE_URI: constant(String[100]) = "{{cookiecutter.base_uri}}"
```